src/oca_github_bot/process.py

```diff
@@ -12,7 +12,9 @@ def call(cmd, cwd):
     return subprocess.call(cmd, cwd=cwd)
 
 
-def check_call(cmd, cwd, log_error=True):
+def check_call(cmd, cwd, log_error=True, extra_cmd=False):
+    if extra_cmd:
+        cmd += extra_cmd.split(",")
```

Comment:
I would put this split in configuration parsing. So here assume extra_cmd is a list.

Reply:
call this `extra_cmd_args`?

Reply:
splitting on whitespace is probably more natural (and we can consider arguments containing spaces are not supported at the moment)

Reply:
> I would put this split in configuration parsing. So here assume extra_cmd is a list.

Indeed.

> call this extra_cmd_args?

Indeed also.

> splitting on whitespace is probably more natural

It was to be consistent with the other args, but you're right. Space are not valid in an arg, and comma could be.